tests/acceptance/test_build.py

```diff
@@ -101,6 +101,28 @@ def test_image_rootfs_extra_space(self, prepared_test_build, bitbake_variables):
         assert(os.stat(built_rootfs).st_size == int(bitbake_variables['MENDER_CALC_ROOTFS_SIZE']) * 1024)
 
 
+    def test_tenant_token(self, prepared_test_build):
+        """Test setting a custom tenant-token"""
+
+        add_to_local_conf(prepared_test_build, 'MENDER_TENANT_TOKEN = "%s"'
+                          %  "authtentoken")
+
+        run_bitbake(prepared_test_build)
+
+        built_rootfs = latest_build_artifact(prepared_test_build['build_dir'], ".ext[234]")
+
+        subprocess.check_call(["debugfs", "-R",
+                                   "dump -p /etc/mender/mender.conf mender.conf", built_rootfs])
+
+        try:
+            output = subprocess.check_output(["debugfs", "-R", "cat mender.conf", built_rootfs])
+            assert(output.find("authtentoken"))
```

Comment:
Did this pass for you? The first `debugfs` call should put the `mender.conf` file on the local file system, so the proper next call would be `output = subprocess.check_output(["cat", "mender.conf"])`.

But I think I instead would restructure this a bit, since it's easy to validate it as JSON. My suggestion:

```
try:
    with open("mender.conf") as fd:
        data = json.load(fd)
    assert data['TenantToken'] == 'authtentoken'
```